pipelines/shopgate.getRootCategories.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": "1",
+  "pipeline": {
+    "id": "shopgate.getRootCategories",
+    "public": true,
+    "input": [],
+    "output": [
+      {
```

Comment:
Should be one line per response property.

Reply:
Done